activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -947,6 +917,8 @@ def serialize(value)
 
         private
 
+        QUOTED_TRUE, QUOTED_FALSE = '1', '0'
```

Comment:
I agree to share same constants with `activerecord/lib/active_record/connection_adapters/mysql/quoting.rb`, because in MySQL world "quoted true" and "quoted false" are unique.